dev-libs/libtypec/libtypec-0.5.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Library to interface with USB Type-c/Power Delivery devices"
+HOMEPAGE="https://github.com/Rajaram-Regupathy/libtypec"
+SRC_URI="https://github.com/Rajaram-Regupathy/libtypec/releases/download/${P}/${P}-Source.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+S="${WORKDIR}/${P}-Source"
+
+src_prepare() {
```

Comment:
You don't need src_prepare, src_compile, or src_install as you're just calling the implementations from the eclass which it exports anyway.